specs/interop/dependency-set.md

```diff
@@ -46,20 +44,6 @@ changes, all possible OP Stack chains in the superchain will be able to exist co
 
 It is a known issue that not all software in the Ethereum can handle 32 byte chain ids.
 
-## Updating the Dependency Set
-
-The `SystemConfig` is updated to have the `SuperchainConfig` address and to manage the dependency set.
```

Comment:
maybe answered later on the PR: why are we not adding that the SuperchainConfig is not in charge of this?

Reply:
We can add a line saying that and linking to the SuperchainConfig section